### PR TITLE
164796902 Add Support for article read time computation

### PR DESCRIPTION
**Pull request by micah-akpan**
#### Description
Users should be able to see the estimated time it will take to read an article
#### Type of change
- [x] New feature (non-breaking change that adds functionality)
#### How Has This Been Tested?
- [x] Integration tests
#### Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have linted my code prior to submission
- [x] New and existing unit tests pass locally with my changes
#### Screenshots:
- Postman (a different REST API client was used):
Screenshot from 2019-04-14 20-46-37
- Flowchart
flow-chart-article-read-time

#### PT
[#164796902](https://www.pivotaltracker.com/story/show/164796902)

### CODE REVIEW

**Comment by ssanusi** [comment]
@micah-akpan LGTM

File: tests/unit/article.test.js [hunk 1]
```diff
@@ -1,37 +1,73 @@
 import 'chai/register-should';
 import { expect } from 'chai';
-import { validateArticle } from '../../src/utils/article';
-import { ARTICLE } from '../mock/article';
+import { generateDummyWords } from '../../src/utils';
+import { validateArticle, computeArticleReadingTime } from '../../src/utils/article';
+import {ARTICLE} from '../mock/article';
```
[the remark this answers is not in the document]
Comment (micah-akpan) [reply tests/unit/article.test.js]: It's been fixed

**Comment by ssanusi** [comment]
LGTM

**Review by Simplemart17 — COMMENTED** [review]
LGTM

**Review by toluola — COMMENTED** [review]
LGTM

**Comment by Raymond-Osy** [comment]
LGTM

**Review by Temmyogunbo — APPROVED** [review]

**Pull request merged**